packages/next/src/client/components/globals/intercept-console-error.ts

```diff
@@ -1,25 +1,22 @@
 import { isNextRouterError } from '../is-next-router-error'
 import { handleClientError } from '../react-dev-overlay/internal/helpers/use-error-handler'
 
+export const originConsoleError = window.console.error
```

Comment:
does this need to be exported?

Reply:
not yet, it was for actually other PR, can be reverted